tests/test_inttensor.py

```diff
@@ -41,3 +41,15 @@ def test_view():
     a.view_(4, -1, 2)
     c_v_ground = IntTensor(np.array([[[9, 3], [1, 0]], [[6, 8], [6, 6]], [[1, 6], [8, 6]], [[5, 0], [2, 0]]]))
     assert(a.equal(c_v_ground))
+
+def test_unfold():
+    a = IntTensor(np.array([[-1, 2, 3, 5], [0, 4, 6, 7], [10, 3, 2, -5]], dtype=np.int32))
+    expected_a = IntTensor(np.array([[[-1, 2, 3, 5], [0, 4, 6, 7]], [[0, 4, 6, 7], [10, 3, 2, -5]]], dtype=np.int32))
+    actual_a = a.unfold(0, 2, 1)
+    assert(actual_a.equal(expected_a))
+
+    expected_a = IntTensor(np.array([[[-1, 2, 3], [0, 4, 6], [10, 3, 2]],
+        [[2, 3, 5], [4, 6, 7], [3, 2, -5]]], dtype=np.int32))
+
+    actual_a = a.unfold(1, 3, 1)
+    assert(actual_a.equal(expected_a))
```

Comment:
Can you please add a unit test for when the step size is greater than 1?  Other than that, PySyft side is looking great!

Reply:
Added unit test for step size of 2